doc/DOCKER_mac_setup.MD

```diff
@@ -0,0 +1,54 @@
+
+#### Move the __New User Folder__ into the __External HD__
+Note: _This is an important step, otherwise, it's probable that when you run the container, it will be installed in your main OS Hard Drive and will run out of space as it validates the Bitcoin blockchain._
+
+1. Open  **Finder** and navigate to your startup drive's **/Users** folder. For most people, this is **/Macintosh HD/Users**. In the **Users**  **folder**, you'll find your user's folder.
+2. On your external Hard Drive, create a folder named **Users**.
+3. Select your user folder and drag it to the external HD **/Users** folder you created. _Because you're using a different drive for the destination, the operating system will copy the data rather than move it. This ok for now but delete it later._
+4. Launch System Preferences again. 
+5. In the  **Users & Groups** click the lock icon in the bottom left corner, then provide an administrator name and password.
+6. From the list of user accounts, right-click on the account whose home folder you moved, and select  **Advanced Options**  from the pop-up menu.
+    _Do not make any changes to Advanced Options except for those noted here. Doing so can cause quite a few unforeseen problems that could lead to data loss or the need to reinstall the operating system._
+7.  In the  **Advanced Options**  sheet, click  **Choose**, located to the right of the  **Home directory**  field.    
+8.  Navigate to the location you moved your home folder to, select the new home folder, and click  **OK**.
+9.  Click  **OK**  to dismiss the  **Advanced Options**  sheet, and then close  **System Preferences**.
+10.  __Restart your Mac__ 
+
+#### Download and install Docker, Kitematic and TOR
+1. Make sure your system fills the [requirements]([https://docs.docker.com/docker-for-mac/install/](https://docs.docker.com/docker-for-mac/install/)) (particularly MacOS Sierra 10.12 or higher. If not, upgrade before proceeding).  
+2. [Download Docker]([https://docs.docker.com/docker-for-mac/install/](https://docs.docker.com/docker-for-mac/install/)) and follow the installation steps.
+3. _Optional_: Download [Kitematic]([https://kitematic.com/) and follow installation instructions. 
+(_This may be system specific but I've found that monitoring the logs with Kitematic was more stable than using the Terminal_).
+4. Install [Tor Browser](https://www.torproject.org/projects/torbrowser.html.en) on the host machine.
+
+## Adjust Docker Settings
+1. Click on the Docker icon (![whale menu](https://docs.docker.com/docker-for-mac/images/whale-x.png)) at the status bar and select __Preferences__.
+2.  Under Disk, click on __Reveal in Finder__ and double check that the disk image is saved under the external HD.
+3. __Adjust Disk__ Image size to 400GB+ and click Apply.
+4. _Optional_: Click __Advanced__ and increase the CPU count, Memory and Swap sizes. Adjusting these will speed up the blockchain validation process 
```

Comment:
suggestion: remove "optional"?
rationale: default 2Gb of RAM seems low for the initial block download